src/stage1/generated.jl

```diff
@@ -225,6 +225,25 @@ function (::∂⃖{N})(f::T, args...) where {T, N}
         end
     end
 end
+function (::∂⃖{N})(::typeof(Core.kwcall), kwargs, f::T, args...) where {T, N}
+    if N == 1
+        # Base case (inlined to avoid ambiguities with manually specified
+        # higher order rules)
+        z = rrule(DiffractorRuleConfig(), KwFunc(f), kwargs, f, args...)
```

Comment:
its basically a copy of the non-kw version of the function, but that is what we want in order to avoid ADing through the kw machinery, if there are no kws if I understand correctly?